infrastructure/store/fake.js

```diff
@@ -4,7 +4,26 @@ class Store {
   constructor() {
     this.products = []
     this.orders = []
-    this.users = []
+    this.users = [
```

Comment:
make sure you have all the fields of makeUser present here in the fake DB for consistency with dynamically created users